### DISCUSSION TITLE
"Rahmen-Überstand" bei Popups

### DISCUSSION THREAD

**Discussion by jp112sdl** [OpenCCU/OpenCCU#1427] [🇩🇪 Ideen]
In _Einstellungen->Systemsteuerung_ steht der Rahmen der Kopf- und Fußzeile bei nahezu allen Popups etwas über.
![grafik](https://user-images.githubusercontent.com/18190139/135207489-7bd60363-d5b1-4d41-a580-62f5d4c846a2.png)

Wird in der `/www/webui/style.css` [Zeile 709](https://github.com/eq-3/occu/blob/f88d81a54e004fb895efbe7ec829c4d1ef28dd47/WebUI/www/webui/style.css#L709)  
```JS
width: 100%;
```
entfernt, passt es.
![grafik](https://user-images.githubusercontent.com/18190139/135207790-3718c280-2914-44bd-a20f-43d9d01acc8c.png)

Negative Auswirkungen konnte ich nicht feststellen.
Aber vielleicht kann das noch jemand mal testen und dann ggf @jens-maus die kleine Änderung mit übernehmen?

EDIT: 
Bei `.popupTitle` das selbe. Dort ist `width` mit 99.4% angegeben und sorgt bei der Condition-Auswahl im Programm dafür, dass der Titel "Wertebereich einstellen" rechts übersteht.
Kommentiere ich `width` aus, ist auch hier alles optisch schick.

**Comment by jens-maus** [comment]
> Aber vielleicht kann das noch jemand mal testen und dann ggf @jens-maus die kleine Änderung mit übernehmen?

Ist doch schon längst (vor 9 Tagen) bereits passiert. Siehe:
https://github.com/jens-maus/RaspberryMatic/commit/c859e2252c390b5aa5e74734899dfa7b117a61c3#diff-6036b5f7a6a3587423463be8d41ba62ff65ee24874206af98edf1ff0da783552

Also einfach mal nen aktuellen nightly snapshot testen und die Popups sollten keinen Überstand mehr haben :)

**Comment by jp112sdl** [comment]
> schon längst (vor 9 Tagen) 

Wer ahnt schon, dass ausgerechnet in der letzten Woche ein "schon immer" mitgeschlepptes CSS-Attribut entfernt wird. ^^ 

Jau, passt so :)

**Reply by jens-maus** [reply]
> > schon längst (vor 9 Tagen)
> 
> Wer ahnt schon, dass ausgerechnet in der letzten Woche ein "schon immer" mitgeschlepptes CSS-Attribut entfernt wird. ^^

Ich sag nur "Zwei Dumme, ein Gedanke!" ;-)
